src/main/java/seedu/address/model/issue/Issue.java

```diff
@@ -0,0 +1,26 @@
+package seedu.address.model.issue;
+
+/**
+ * Temporary stub class.
+ */
+public class Issue {
+    public Object getDeadline() {
+        return null;
+    }
+
+    public Object getDescription() {
+        return null;
+    }
+
+    public Object getPriority() {
+        return null;
+    }
+
+    public Object getProject() {
```

Comment:
A quick note: we might want to discuss next meeting if an Issue should have a Project tied to it as opposed to the other way around?